lua/packer/plugin_types/git.lua

```diff
@@ -127,12 +127,12 @@ local handle_checkouts = function(plugin, dest, disp)
       end
     end
 
-    if plugin.branch or plugin.tag then
+    if plugin.branch or (plugin.tag and not opts.preview_updates) then
```

Comment:
I'll test this, but: can this not cause bugs with `preview_updates` enabled and changing a manually-specified tag (i.e., not `'*'`)?

Reply:
I don't think so since this will anyway be called once we continue (`c`) after the preview. But let me know if it doesn't work as intended/expected.

Reply:
Seems to work, thanks.